lib/secure_headers/headers/content_security_policy.rb

```diff
@@ -154,10 +154,10 @@ def reject_all_values_if_none(source_list)
     # e.g. *.github.com asdf.github.com becomes *.github.com
     def dedup_source_list(sources)
       sources = sources.uniq
-      wild_sources = sources.select { |source| source =~ STAR_REGEXP }
+      wild_sources = sources.select { |source| source =~ DOMAIN_WILDCARD_REGEX }
```

Comment:
Could you explain the reasoning behind this?

As far as I Can tell, it should be fine to remove `example.org:443` if `example.org:*` is present.